### PR TITLE
OSDOCS#10757: Release notes for removal of Alibaba support in OCP

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-10757

Link to docs preview: https://76754--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-installation-and-update-alibaba_release-notes

https://76754--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-technology-preview-tables_release-notes --> Installation Technology Preview features

**TODO: Link to new Alibaba docs before merging**

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by jianli-wei** [comment]
@snarayan-redhat it looks like the links to doc preview doesn't have the changes, any suggestion? Thanks!

**Comment by snarayan-redhat** [comment]
> @snarayan-redhat it looks like the [links](https://76754--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html) to doc preview doesn't have the changes, any suggestion? Thanks!

You should be able to see now. The link to new Alibaba docs is still pending, which I will update before merging the PR.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -73,6 +73,10 @@ With this release, you can now install {op-system} to iSCSI boot devices. Multip
 In {product-title} {product-version}, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on AWS. There are several additional required permissions as a result of this change. For more information, see xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[Required AWS permissions for the IAM user].
 
 Additionally, SSH access to control plane and compute machines is no longer open to the machine network, but is restricted to the security groups associated with the control plane and compute plane machines.
+[id="ocp-4-16-installation-and-update-alibaba_{context}"]
+==== Installation on Alibaba Cloud by using Assisted Installer (Technology Preview)
+
+ With this release, {product-title} installation program no longer supports the installer-provisioned installation on Alibaba Cloud platform. You can install a cluster on Alibaba Cloud by using Assisted Installer, which is currently a Technology Preview feature. For more information, see <xref to Alibaba docs in OCP>.
```
Comment (jldohmann) [inline release_notes/ocp-4-16-release-notes.adoc:79]:
```suggestion
 With this release, the {product-title} installation program no longer supports installer-provisioned installation on the Alibaba Cloud platform. You can install a cluster on Alibaba Cloud by using Assisted Installer, which is currently a Technology Preview feature. For more information, see <xref to Alibaba docs in OCP>.
```

**Review by jldohmann — COMMENTED** [review]
Just a minor comment below, but I also noticed the preview doc shows the release note entry as if it were a code block. i'm not sure why, I don't see any backticks in the file changes

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply release_notes/ocp-4-16-release-notes.adoc]: Done. 
For the broken preview, there was a space introduced in the beginning of the sentence which caused the whole block to be rendered as a code. It's strange and took me some time to figure that out. :D

**Pull request merged**